docs/_pages/advanced_numerical.rst

```diff
@@ -0,0 +1,145 @@
+Advanced Numerical Libraries
+============================
+
+This page introduces you to a set of powerful Python libraries for advanced numerical computing. Python has libraries for machine learning, model fitting, statistics, network calculations, and much more! 
+
+scikit-learn
+------------
+
+scikit-learn is a powerful Python library for machine learning. It contains a wide array of useful tools for classification, regression, clustering, dimensionality reduction, and much more. Implementations are available for most standard machine learning algorithms, such as Support Vectro Machines and Decision Trees. If you're planning to implement machine learning in your analyses, then you will want to install scikit-learn. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install scikit-learn
+
+scikit-image
+------------
+
+scikit-image is a very useful Python library for handling image data. It's a powerful tool for image processing that allows users to identify objects, filter images, mainipulate color channels, and a wide variety of other tasks. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install scikit-image
+
+TensorFlow
+----------
+
+TensorFlow is a versatile library designed for implementations of deep learning algorithms. If you're looking to use deep neural networks on your data, for example a large-scale image classification problem, then TensorFlow will likely be useful. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install tensorflow
+
+Theano
+----------
+
+Theano is a library geared toward efficient computations on multidimensional arrays. It also supports implementation of code on GPUs. Theano is useful for implementing deep learning in Python.
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install theano
+
+Keras
+----------
+
+Keras is another Python library for machine learning using neural networks. It is capable of interacting with other machine learning libraries, including TensorFlow and Theano. Keras runs on CPUs and GPUs and is designed for fast implementation of neural networks.
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install keras
+
+LmFit
+-----
+
+LmFit is a powerful library for curve-fitting in Python. It implements several methods for non-linear least-squares minimization and introduces some very useful classes. The LmFit Parameters class offers a clean way of handling model paramters while fitting. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install lmfit
+
+emcee
+-----
+
+emcee is a Python library built for efficient Bayesian computations. It implements the Affine Invariant Markov chain Monte Carlo (MCMC) Ensemble sampler to reconstruct posterior probability distributions for model parameters. If you're looking to implement a Bayesian fit in your analysis this libary will be a useful tool. 
```

Comment:
- Second sentence is maybe too jargony?
- How about: "emcee is a Python library for efficiently estimating probability distributions. It uses efficient MCMC sampling strategy that is often used to approximate posterior distributions in Bayes Theorem. 

It uses a effici

Reply:
:+1: